tests/reference/c-expr7-bb2692a.stdout

```diff
@@ -1,6 +1,6 @@
-#include <complex.h>
-#include <inttypes.h>
-#include <math.h>
+#include "complex.h"
+#include "inttypes.h"
+#include "math.h"
```

Comment:
I think this change is a problem, it should use `<`, not `"`.

Reply:
Yep, thanks. Reverted that.